demos/graspTheBall/main.yml

```diff
@@ -13,8 +13,8 @@ x-yarp-base: &yarp-base
     - ARM
     - YARP_ROBOT_NAME
   volumes:
-    - "/tmp/.X11-unix:/tmp/.X11-unix:rw"
-    - "${XAUTHORITY}:/root/.Xauthority:rw"
+    - "/tmp/.X11-unix:/tmp/.X11-unix"
+    - "${XAUTHORITY}:/root/.Xauthority"
     - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
     - "${APPSAWAY_DEMOREDBALL_CONTEXT}:/usr/local/share/ICUBcontrib/contexts/demoRedBall/"
```

Comment:
isn't this volume supposed to share some files to be used in the real robot? Or is it only used inside the demo?

Reply:
I think the first two should not be rw, but the last has to. I updated it